agent/consul/state/peering.go

```diff
@@ -7,12 +7,13 @@ import (
 	"strings"
 
 	"github.com/golang/protobuf/proto"
+	"github.com/hashicorp/go-memdb"
+
```

Comment:
weird

Reply:
It got moved because it's not `hashicorp/consul` 😆